### DISCUSSION TITLE
Adding external code from Github/Gitlab into material-mkdocs using "Snippets" extension

### DISCUSSION THREAD

**Discussion by imflash217** [squidfunk/mkdocs-material#3318] [Q&A] [answered]
Hi,
I am trying to add an external file from my github/gitlab repo into my mkdocs project using "Snippets" Extension but unfortunately it doe not accept it.
How do we achieve this.
Thanks

**Comment by squidfunk** [comment, marked as answer]
IIRC, Snippets does not support external files. You must download them before.

**Reply by facelessuser** [reply]
There's an open issue to add downloading external snippets that I've been considering. I'm not sure if/when I'm doing it yet, but you can follow this issue for future updates: https://github.com/facelessuser/pymdown-extensions/issues/1350.

**Reply by imflash217** [reply]
Thanks @squidfunk @facelessuser .
